tools/python/boutiques/tests/test_prepare.py

```diff
@@ -101,10 +101,10 @@ def test_prepare_sing_image_does_not_exist(self):
         with pytest.raises(ExecutorError) as e:
             bosh.execute("prepare",
                          os.path.join(example1_dir,
-                                      "example1_sing.json"),
+                                      "example1_sing_crash_pull.json"),
```

Comment:
Good point! The test was passing before due to `test` though. At some point we should mock that.

Reply:
to clarify should we leave it as _sing_crash_pull or revert to _sing? 
I figured that giving an invalid container would be scenario that this test seeks to evalute, let me know if this is right